backend/app/tests/features/briefing_phase.feature

```diff
@@ -0,0 +1,25 @@
+@briefingphase
+Feature: Briefing phase of the game
+  
+  Testing briefing phase for API: https://coldwar-api.onrender.com/docs
+
+  Scenario: user goes into briefing phase with valid 'access_token' and without 'Analyst' card ability
+    Given user using valid 'access_token'
+    When user goes into 'Briefing' phase
```

Comment:
Give me a more info what happening - we test api or abstract things from your mind?

```gherkin
When user request <...> and get response <...>
  Then user request <...> and get response <...>
  And user request <...> and get response <...>
  
When user request <...> with <...> and get response <...>
  Then response contain <...>
  And response contain <...>
````

and something like this (this is example). Dont provoke me play a gamle - what does mean 'user goes into 'Briefing' phase`